docs/en/observability/install-apm.asciidoc

```diff
@@ -0,0 +1,31 @@
+[chapter, role="xpack"]
+[[observability-instrument-apps]]
+= Instrument applications
+
+Elastic APM monitors software services and applications in real time, collects 
+unhandled errors and exceptions, and automatically picks up basic host-level 
+metrics and agent specific metrics.
+
+include::{apm-repo-dir}/guide/install-and-run.asciidoc[]
```

Comment:
Step 0 (Install and run), Step 1 (Install ES), Step 2 (Install Kib), and Step 3 (Install APM Server) don't seem like they belong on this page.  For a page called "instrument applications", I think we should be starting with step 4 (Install APM Agents).

You could add tagged regions to this file in the APM documentation, just make sure to exclude the titles so that you can set your own step numbers in this book

Reply:
I'm thinking something like: 

```
Instrument applications
Elastic APM monitors software services and applications in real time, collects unhandled errors and exceptions, and automatically picks up basic host-level metrics and agent specific metrics.

NOTE: If you haven’t already, you need to install Elasticsearch for storing and searching your data, Kibana for visualizing and managing it, and APM Server. For more information, see Get started.

Step 1. Install APM Agents
Step 2. Configure APM
Step 3. View your data in Kibana

Reply:
Thanks! I created the tagged regions.